icons/vercel_light.svg

```diff
@@ -1 +1,3 @@
-<svg version="1.1" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><g transform="matrix(.49799 0 0 .49799 -309.62 -27.09)" style="fill-rule:evenodd;fill:none;stroke:#455a64"><g transform="translate(628,63)" style="fill:none;stroke:#455a64"><g style="fill:none;stroke-width:2;stroke:#455a64"><polygon points="0 31.997 17.844 .046641 35.689 31.997" style="fill:none;stroke:#455a64"/></g><path d="m34.085 31.59-16.851-30.359" style="fill:none;stroke-linecap:square;stroke:#455a64"/><path d="m34.851 31.59-16.851-30.359" style="fill:none;stroke-linecap:square;stroke:#455a64"/><path d="m33.319 31.59-16.085-29.538" style="fill:none;stroke-linecap:square;stroke:#455a64"/><path d="m32.553 31.59-15.319-28.718" style="fill:none;stroke-linecap:square;stroke:#455a64"/><path d="m16.468 2.8718 16.851 28.718" style="fill:none;stroke-linecap:square;stroke:#455a64"/><path d="m31.787 31.59-15.319-27.897" style="fill:none;stroke-linecap:square;stroke:#455a64"/><path d="m28.723 25.026-6.1277-10.667" style="fill:none;stroke-linecap:square;stroke:#455a64"/></g></g></svg>
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please switch the icons: the light icons should be dark to have a good contrast to the light background and the other icon which is used on a dark background theme should have a lighter color so that it has a good contrast on a dark background. Otherwise we would have a dark icon on a dark background which will not look good!

Reply:
Done 👍